src/display/annotation_layer.js

```diff
@@ -239,6 +239,40 @@ class AnnotationElement {
     return container;
   }
 
+  /**
+   * Create areas for the quadPoints.
+   *
+   * @private
+   * @memberof AnnotationElement
+   */
+  _createAreas(ignoreBorder = false) {
```

Comment:
`_createQuadrilaterals` (that's what the specification calls the "areas"), so let's rename all "area" occurrences to "quadrilateral".